mama/c_cpp/src/c/transport.c

```diff
@@ -1907,53 +1970,6 @@ mamaTransportImpl_preRecapCacheEnabled (mamaTransport transport)
     return 0;
 }
 
-/* Process an advisory message and invokes callbacks
- *                    on all listeners.
- * @param transport The transport.
- * @param cause Cause for the advisory.
- * @param platformInfo Additional bridge specific information that will
- *                     be passed to all listeners.
- */
-void
-mamaTransportImpl_processAdvisory (mamaTransport transport,
```

Comment:
Actually, put this function back in again - I know I said it wasn't used (and it isn't) but I couldn't really be bothered arguing the toss with The Community.  Since the event is QUALITY you can actually differentiate it from the DISCONNECT events in the event hook anyway